gems/smithy/lib/smithy.rb

```diff
@@ -7,11 +7,9 @@
 require_relative 'smithy/forge'
 require_relative 'smithy/model'
 require_relative 'smithy/plan'
-require_relative 'smithy/polish'
-require_relative 'smithy/polishes'
+require_relative 'smithy/rivet'
```

Comment:
I like the integration of the polish/post-process into weld, but why the rename to rivet?  I like name weld.

Though conceptually both are about equally meaningful/applicable to the functionality, so I'm fine with the rename just curious why.